var/spack/repos/builtin/packages/caffe/package.py

```diff
@@ -40,10 +40,10 @@ class Caffe(CMakePackage):
     depends_on('protobuf')
     depends_on('glog')
     depends_on('gflags')
-    depends_on('hdf5')
+    depends_on('hdf5 +hl +cxx')
 
     # Optional dependencies
-    depends_on('opencv@3.2.0+core+highgui+imgproc', when='+opencv')
+    depends_on('opencv@3.2.0+core+highgui+imgproc~vtk~videoio~gtk~python', when='+opencv')
```

Comment:
`caffe` doesn't work if you link to `opencv+vtk...` or you just want to change the default?

Reply:
openvc@3.2.0+vtk... did not build.
I'm backport to build openvc.